### DISCUSSION TITLE
Allow customization of /list output

### DISCUSSION THREAD

**Discussion by Bobcat00** [EssentialsX/Essentials#4611] [Ideas and Feature Suggestions] [answered]
When a player has a nickname, the output of the /list command includes the player's real name in parenthesis, in white. However, that cannot be customized to use different colors or for that matter  something other than parentheses. The format appears to be hardcoded in PlayerList#listUsers.

I suggest the _(playername)_ part be customizable like the rest of the message.

![list_output](https://user-images.githubusercontent.com/3617206/140448488-99dcd571-2f54-4f63-a065-aac49b47fe30.png)

**Comment by JRoy** [comment, marked as answer]
Added in https://github.com/EssentialsX/Essentials/commit/e4cf965baeb6de3de1aad6d28277b5bb99c37a53
